### ISSUE TITLE
decreasing input_size gives very low detection accuracy

### ISSUE THREAD

**Issue by hadizand** [google/automl#779]
I trained D0, D1, D2 with their default input sizes (512, 640, 768). The train models work very well. When I decreased the input_size flag to 300, the new train models do not detect well. I am using a custom dataset, by decreasing the input size, obtained models could not detect well (about half of the objects were not detected). 
This is my config file:
num_classes: 2
label_map: {1: nail}
use_keras_model: False
moving_average_decay: 0
image_size: 300

Also, I trained models with input_size=256 which is a multiple of 128 (as parer suggested), but the performance is not good.
I understand that reducing the input size can affect the accuracy, but is it normal to drop the performance by 50%?
In my experiments, I used the same hyperparameters for default and decreased input_size.

**Comment by mingxingtan** [comment]
A few thoughts:

- 1. you should change anchor_scale to 2 (from the default 4):  the current anchoar_scale is 4, meaning the minimal box size is 4 * 2**3 = 32x32 and max box size is 4 * 2**7 = 512x512.  If you use image size 300, I recommend changing anchor_scale to 2.

- 2. you can just use 300x300:  the multiple of 128 in the paper is due to a limitation of our initial code, now you can use any image size.

- 3. What do you mean by performance drop by 50%? 

In addition, I am not sure whether you need to tune other hyperparameters (like jitter scaling and epochs)

**Comment by hadizand** [comment]
> A few thoughts:
> 
> * 1. you should change anchor_scale to 2 (from the default 4):  the current anchoar_scale is 4, meaning the minimal box size is 4 * 2**3 = 32x32 and max box size is 4 * 2**7 = 512x512.  If you use image size 300, I recommend changing anchor_scale to 2.
> * 1. you can just use 300x300:  the multiple of 128 in the paper is due to a limitation of our initial code, now you can use any image size.
> * 1. What do you mean by performance drop by 50%?
> 
> In addition, I am not sure whether you need to tune other hyperparameters (like jitter scaling and epochs)

Thank you very much for your clarification.  I applied your comments, it works.
